shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/viewModel/MapViewModel.kt

```diff
@@ -193,13 +194,18 @@ class MapViewModel(
             }
         }
 
+        LaunchedEffect(Unit) {
+            // Route card data is sent through a separate StateFlow rather than the event flow
+            // because frequent updates were causing buffer overflows.
+            routeCardDataUpdates.collectLatest { routeCardData = it }
```

Comment:
It might even work to `val routeCardData = routeCardDataUpdates.collectAsState()` and skip this `LaunchedEffect` entirely.